src/components/datePicker/datePicker.component.html

```diff
@@ -1,7 +1,7 @@
 <div [ngSwitch]="isMobile"
      [class.invalid]="invalid">
 	<div *ngSwitchCase="true">
-		<input type="date" #input [disabled]="disableInput || disablePicker" [placeholder]="placeholder" [value]="mobileFormattedDate" (change)="setDate($event.target.value)" [tabindex]="inputTabIndex" [min]="min" [max]="max" />
+		<input type="date" #input [disabled]="disableInput || disablePicker" [placeholder]="placeholder" [value]="mobileFormattedDate" (change)="setDate($any($event.target).value)" [tabindex]="inputTabIndex" [min]="min" [max]="max" />
```

Comment:
Not sure how I feel about using `$any` here.

Generally, Typescript's type-checking saves our skin a lot of times by alerting when the types don't quite line up. Casting values to `any` opts-out of that functionality, which could hurt us down the line.

From what I can tell, this change was likely needed because angular compilation was throwing an error saying that the `$event.target.value` type didn't exist. Makes sense - Typescript gets very finicky when it comes to DOM node types. There are multiple solutions to this problem - I might have added a new `setDate` function that accepts an argument of whatever type `$event` is & passes it to the original `setDate` function. But you could use your own discretion.